R/summarize_weighted_production.R

```diff
@@ -303,7 +302,7 @@ add_technology_share_target <- function(data) {
 
 check_zero_initial_production <- function(data) {
   companies_with_zero_initial_production <- data %>%
-    group_by(.data$technology, .data$name_abcd, .data$year) %>%
+    group_by(.data$technology, .data$name_abcd) %>%
```

Comment:
Apologies for this, it is a VERY random change in a bug I found in one of the checks for this function. 
I probably could have opened a separate PR for this, but it is truly just a change to a "check" (not even a change to the function logic itself), so I thought ok to just deal with it. 

The bug was identified when trying to render the `pkgdown` vignettes after opening this PR.